testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/component/SampleExternalWebView.kt

```diff
@@ -25,3 +29,18 @@ class SampleWebViewClient(private val miniAppExternalUrlLoader: MiniAppExternalU
     override fun shouldOverrideUrlLoading(view: WebView?, url: String?): Boolean =
         miniAppExternalUrlLoader.shouldOverrideUrlLoading(url ?: "")
 }
+
+class SampleWebChromeClient(val context: Context) : WebChromeClient() {
+
+    override fun onCreateWindow(
+        view: WebView?,
+        isDialog: Boolean,
+        isUserGesture: Boolean,
+        resultMsg: Message?
+    ): Boolean {
+        // redirect to default browser when there is window.open / _blank type anchor
```

Comment:
so finally we decided to make this no action or open default browser? @corycaywood

Reply:
I think probably this is okay - to me it seems like the more reasonable behavior for `window.open` and `_blank` to be sent to the default browser. 

But we would also need to change iOS to match this. Is this possible @Climbatize @ts-leojoseph-r ?

Reply:
I missed your mention. I think it's quite possible, at least for blank targets